src/mlj_docstrings.jl

```diff
@@ -0,0 +1,146 @@
+
+const CATBOOST_DOCS_LINK = "https://catboost.ai/en/docs/"
+const CATBOOST_PARAMS_DOCS_LINK = "https://catboost.ai/en/docs/references/training-parameters/"
+
+"""
+$(MMI.doc_header(CatBoostRegressor))
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X, y)
+
+where
+
+- `X`: any table of input features (eg, a `DataFrame`) whose columns
+  each have one of the following element scitypes: `Continuous`,
+  `Count`, `Finite`, `Textual`; check column scitypes with `schema(X)`.
+  `Textual` columns will be passed to catboost as `text_features`,
+  `Multiclass` columns will be passed to catboost as `cat_features`, and
+  `OrderedFactor` columns will be converted to integers.
+
+- `y`: the target, which can be any `AbstractVector` whose element
+  scitype is `Finite`; check the scitype with `scitype(y)`
+
+Train the machine with `fit!(mach, rows=...)`.
+
+
+# Hyper-parameters
+#TODO: Document hyper-parameters
+https://catboost.ai/en/docs/concepts/python-reference_catboostclassifier#parameters
+
+# Operations
+
+- `predict(mach, Xnew)`: return predictions of the target given new
```

Comment:
It seems the two method descriptions are switched:

```suggestion
- `predict(mach, Xnew)`: return probabilistic predictions of the target given new
```

I don't see that you need to document `predict_mode` (we haven't done so elsewhere) but won't insist you don't.

Reply:
I removed it